arduino/redcar_control/redcar_control.ino

```diff
@@ -8,11 +8,11 @@
 
 double left_setpoint = 0; // declare ALL the variables
 double right_setpoint = 0;
-double setpoint = 183;
+double setpoint = 319;
```

Comment:
What is the significance of this initialized value and the one on [line 15](https://github.com/SJSU-AD/FusionAD/pull/117/files#diff-b40eb6cc4fa01e8b961e02bca1758dd5R15)?

Reply:
Resolved the redundancy, deleted the second variable -- "incoming".

Reply:
Resolved the redundancy, deleted the second variable -- "incoming".